content/en/agent/kubernetes/operator_configuration.md

```diff
@@ -0,0 +1,214 @@
+|--------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| agent.additionalAnnotations                                                                                | AdditionalAnnotations provide annotations that will be added to the Agent Pods.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| agent.additionalLabels                                                                                     | AdditionalLabels provide labels that will be added to the cluster checks runner Pods.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| agent.apm.enabled                                                                                          | Enable this to enable APM and tracing, on port 8126 ref: https://github.com/DataDog/docker-dd-agent#tracing-from-the-host                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
same comment about making these actual clickable URLs. You also use periods inconsistently in this table too.